en/firefox/accounts.ftl

```diff
@@ -5,74 +5,49 @@
-firefox-accounts-get-a-lookout-for = Get a lookout for data breaches.
+# This is followed by a list of things you can do with your Firefox account
+firefox-accounts-sign-in-to = Sign in to your { -brand-name-firefox-account } to:
 
 # Obsolete string
-firefox-accounts-share-large-files = Share large files without prying eyes.
-firefox-accounts-get-it-all-on-every = Get it all on every device, without feeling trapped in a single operating system.
-firefox-accounts-and-get-it-all-on = And get it all on every device, without feeling trapped in a single operating system.
-firefox-accounts-get-the-respect-you = Get the respect you deserve.
-
-firefox-accounts-mozilla-vpn = { -brand-name-mozilla-vpn }
-firefox-accounts-whole-device-protection = Whole-device protection on every device
+firefox-accounts-meet-our-family-of = Meet our family of privacy-first products.
 
 # Variables:
-#   $promise (url) - link to https://www.mozilla.org/firefox/privacy/
-firefox-accounts-youll-always-get-the = You’ll always get the truth from us. Everything we make and do honors our <a href="{ $promise }">Personal Data Promise</a>:
+#   $send (url) - link to https://blog.mozilla.org/en/products/firefox/firefox-tips/firefox-secret-tips/#send-tabs
+firefox-accounts-send-a-tab = <a href="{ $send }">Send a tab</a> to a different device
 
-# Line breaks for visual formatting.
-firefox-accounts-take-less-keep-it = Take less.<br> Keep it safe.<br> No secrets.
+# Variables:
+#   $password (url) - link to https://www.mozilla.org/firefox/features/password-manager/
+firefox-accounts-save-and-sync = <a href="{ $password }">Save and sync passwords</a>
+firefox-accounts-get-email-alerts = Get email alerts when your info appears in a known data breach
 
-firefox-accounts-get-the-knowledge = Get the knowledge to keep you safe.
-firefox-accounts-learn-everything-you = Learn everything you need to know (but don’t yet) about staying smart and safe online, from some of the world’s foremost experts.
-firefox-accounts-and-be-part-of-protecting = And be part of protecting the internet for future generations.
-firefox-accounts-help-us-build-a-better = Help us build a better { -brand-name-firefox } for all.
-firefox-accounts-get-into-the-open = Get into the open source spirit by test-driving upcoming products.
+# Obsolete string
+firefox-accounts-see-if-youve-been = See if you’ve been involved in an online data breach.
+firefox-accounts-protect-your-identity = Protect your identity with secure phone and email masking
+firefox-accounts-keep-your-passwords = Keep your passwords protected and portable.
+firefox-accounts-get-a-lookout-for = Get a lookout for data breaches.
+firefox-accounts-travel-the-internet = Travel the internet with protection, on every device.
+firefox-accounts-encrypt-your = Encrypt your network activity and hide your IP address
+firefox-accounts-we-never = We never log, track, or share your network data
```

Comment:
> It's a list of different things you can do with a Firefox account. We should add a comment.

And this sentence doesn't fit at all in this pattern, making the translation quite a mess.

`Sign in to your { -brand-name-firefox-account } to:` -> `We never log, track, or share your network data` 🤔